### PR TITLE
Add scorecards analysis

### PR DESCRIPTION
**Pull request by oh6hay**
Github Action for Scorecard
https://github.com/ossf/scorecard

The Scorecard provides a (somewhat opinionated) view on the security posture of the repository -- we (the product security team) would like to adopt this on our key repositories to identify possible security improvements

### CODE REVIEW

File: .github/workflows/scorecards-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+name: Scorecards supply-chain security
+on:
+  # Only the default branch is supported.
+  branch_protection_rule:
+  schedule:
+    - cron: '22 14 * * 1'
+  push:
+    branches: [ master ]
```
Comment (enahum) [inline .github/workflows/scorecards-analysis.yml:8]: Do we want to check master which is the current mobile app or the gekidou branch which is the ipcoming mobile v2?

Comment (oh6hay) [reply .github/workflows/scorecards-analysis.yml:8]: I think you can pick either if you want to -- currently we are adding the scorecards to get a baseline on how things are looking. If we pick one here for the github action, the scorecard thing can be manually and locally run for the other branch as well.

For now master is ok imho as it's out and in production.

**Review by enahum — APPROVED** [review]
Lgtm

File: .github/workflows/scorecards-analysis.yml (hunk 1 above)
Comment (jupenur) [reply .github/workflows/scorecards-analysis.yml:8]: Yeah I agree for this it probably makes more sense to go with master.

**Review by jupenur — APPROVED** [review]
LGTM!

**Pull request merged**